cfgov/v1/atomic_elements/organisms.py

```diff
@@ -954,7 +954,7 @@ class Meta:
         classname = 'block__flush'
 
     class Media:
-        js = ['video-player.js']
+        js = ['featured-content-module.js']
```

Comment:
Out of scope for this PR, but it would be nice if we could figure out how to not load this JS if a given FCM doesn't have a video in it, only a static image.

Reply:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/5162/commits/868a228a15a0764b43eb6f4e9609c93693d39de6 to bail out when there is no video present.